docs/design/admission_control_limit_range.md

```diff
@@ -73,8 +73,8 @@ type LimitRangeItem struct {
   Min ResourceList `json:"min,omitempty" description:"min usage constraints on this kind by resource name"`
   // Default resource limits on this kind by resource name
   Default ResourceList `json:"default,omitempty" description:"default resource limits values on this kind by resource name if omitted"`
-  // DefaultRequests resource requests on this kind by resource name
-  DefaultRequests ResourceList `json:"defaultRequests,omitempty" description:"default resource requests values on this kind by resource name if omitted"`
+  // DefaultRequest resource requests on this kind by resource name
+  DefaultRequest ResourceList `json:"defaultRequest,omitempty" description:"default resource requests values on this kind by resource name if omitted"`
```

Comment:
I agree on DefaultLimits, but did not want to make that change until v2
API.

On Wednesday, August 12, 2015, Brian Grant <notifications@github.com> wrote:

> In docs/design/admission_control_limit_range.md
> <https://github.com/kubernetes/kubernetes/pull/12492#discussion_r36936897>
> :
>
> > @@ -73,8 +73,8 @@ type LimitRangeItem struct {
> >    Min ResourceList `json:"min,omitempty" description:"min usage constraints on this kind by resource name"`
> >    // Default resource limits on this kind by resource name
> >    Default ResourceList `json:"default,omitempty" description:"default resource limits values on this kind by resource name if omitted"`
> > -  // DefaultRequests resource requests on this kind by resource name
> > -  DefaultRequests ResourceList `json:"defaultRequests,omitempty" description:"default resource requests values on this kind by resource name if omitted"`
> > +  // DefaultRequest resource requests on this kind by resource name
> > +  DefaultRequest ResourceList `json:"defaultRequest,omitempty" description:"default resource requests values on this kind by resource name if omitted"`
>
> Technically, Defaults should be DefaultLimits, or Defaults should be of
> type ResourceRequirements.
>
> DefaultRequest is fine.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/12492/files#r36936897>.
>